src/utils/api.js

```diff
@@ -1,26 +1,47 @@
-import {shouldUseProxyUrl, proxyUrl, fhirUrl} from '../config';
+import {shouldUseProxyUrl, proxyUrl, fhirUrl, shouldUseCookie} from '../config';
 import {logErrors, replaceLocalhost} from '../utils/common';
 import store from '../store';
 
+const getHeaders = headers => {
+  let allHeaders = {
+    'Cache-Control': 'max-age=3600',
+    Accept: 'application/fhir+json;charset=utf-8',
+    ...headers,
+  };
+  const selectedServer = store.getState().app.selectedServer;
+  const requiresAuth = selectedServer.authType !== 'NO_AUTH';
+  if (requiresAuth && !shouldUseProxyUrl(selectedServer.url)) {
+    const token = store.getState().app.token;
+    allHeaders['Authorization'] = `Basic ${token}`;
+  }
+  return allHeaders;
+};
+
+const setCredentials = url => {
```

Comment:
Shouldn't this return an object like `{credentials: 'include'}`? Because it looks like right now the value of `credentials` on line 75 is just a string `include`. I'm remembering what we manually entered in through chrome dev tools and also looking at the example here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch